ODPi-Runtime.md

```diff
@@ -154,7 +150,7 @@ The following environment variables are noted by this spec:
 | **[HADOOP_EM3]** | MAPRED_LIB_JARS_DIR | Relative Dir | Additional Apache Hadoop MapReduce jar dependencies |
 
 
--   The content of the `*_DIR` directories SHOULD be the same as the ODPi Reference Implementation and the Apache Hadoop distribution of the appropriate platform.  In a future release, this will become a MUST.
+-   The content of the `*_DIR` directories SHOULD be the same as the ODPi Reference Implementation and the binary packages made available by the upstream projects.  In a future release, this will become a MUST.
```

Comment:
Personally that addition of "...upstream projects." makes it confusing to me. I'd drop that part completely.